### PR TITLE
fix(vital-demo): Add Homepage Content & minor fixes

### PR DESCRIPTION
**Pull request by hvanyo**
### Changes

Content: 

- Added Content to Homepage
- Fixed Styleguide link in menu
- Fixed card link on /styleguide page.

Code: 

- Restricted Topslot on generic template to constrain to 2xl:  
- Moved breadcrumb styling to correct place (was a todo) so it takes effect.

### Testing:

- Verify Homepage Content
- Verify Styleguide link in menu
- Verify card link on /styleguide page
- Verify if browser is open larger than 1536 than top slot is contained to 1536.)
- Verify breadcrumbs should get the font styles: text-vital-primary-body-copy font-normal text-m-xs lg:text-xs

### CODE REVIEW

File: sites/vital-demo/src/data/pages/main-content$90b08540-e809-11ec-b8d2-4bbfdb166c16$description.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "type": "paragraph",
+    "children": [
+      {
+        "text": "Our soap has natural moisturizers to strenthen your skin's moisture barrier."
```
Comment (dewen) [inline sites/vital-demo/src/data/pages/main-content$90b08540-e809-11ec-b8d2-4bbfdb166c16$description.json:6]: strenthen => strengthen

File: sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "src": "/images/pages/783f2677b3d14d9221859a1347a7d120/soap1smaller.jpeg",
+  "alt": "Fragrance Free Soap ",
```
Comment (dewen) [inline sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json:3]: whitespace after `Soap`

File: sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  "title": "Fragrance Free Soap ",
```
Comment (dewen) [inline sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json:4]: extra whitespace

File: sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$title.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "text": "Fragrance Free Soap&nbsp;"
```
Comment (dewen) [inline sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$title.json:2]: remove `&nbsp;` if added by accident

File: sites/vital-demo/src/data/pages/main-content$90b08540-e809-11ec-b8d2-4bbfdb166c16$description.json (hunk 1 above)
Comment (hvanyo) [inline sites/vital-demo/src/data/pages/main-content$90b08540-e809-11ec-b8d2-4bbfdb166c16$description.json:6]:
```suggestion
        "text": "Our soap has natural moisturizers to strengthen your skin's moisture barrier."
```

File: sites/vital-demo/src/data/pages/main-content$96974b10-e809-11ec-b8d2-4bbfdb166c16$eyebrow.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "text": "AdditivE Free"
```
Comment (hvanyo) [inline sites/vital-demo/src/data/pages/main-content$96974b10-e809-11ec-b8d2-4bbfdb166c16$eyebrow.json:2]:
```suggestion
  "text": "Additive Free"
```

File: sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json (hunk 2 above)
Comment (hvanyo) [inline sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json:3]:
```suggestion
  "alt": "Fragrance Free Soap",
```

File: sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json (hunk 3 above)
Comment (hvanyo) [inline sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$image.json:4]:
```suggestion
  "title": "Fragrance Free Soap",
```

File: sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$title.json (hunk 4 above)
Comment (hvanyo) [inline sites/vital-demo/src/data/pages/main-content$a618c8e0-e807-11ec-b8d2-4bbfdb166c16$title.json:2]:
```suggestion
  "text": "Fragrance Free Soap"
```

**Review by dewen — APPROVED** [review]

**Pull request merged**